src/main/java/org/jenkinsci/plugins/workflow/libs/LibraryCachingConfiguration.java

```diff
@@ -71,62 +67,40 @@ public Boolean isRefreshEnabled() {
     public String getExcludedVersionsStr() {
         return excludedVersionsStr;
     }
+
+    @CheckForNull
     public String getIncludedVersionsStr() {
-        if(StringUtils.isBlank(includedVersionsStr)){
-            return null;
-        }
-            return includedVersionsStr;
+        return Util.fixEmptyAndTrim(includedVersionsStr);
     }
 
     @DataBoundSetter
     public void setIncludedVersionsStr(String includedVersionsStr) {
-        this.includedVersionsStr = includedVersionsStr;
+        this.includedVersionsStr = Util.fixEmptyAndTrim(includedVersionsStr);
     }
 
-    private List<String> getExcludedVersions() {
-        if (excludedVersionsStr == null) {
-            return Collections.emptyList();
+    private static Stream<String> split(@CheckForNull String list) {
+        if (list == null) {
+            return Stream.empty();
         }
-        return Arrays.asList(excludedVersionsStr.split(VERSIONS_SEPARATOR));
-    }
-
-    private List<String> getIncludedVersions() {
-        if (includedVersionsStr == null) {
-            return Collections.emptyList();
-        }
-        return Arrays.asList(includedVersionsStr.split(VERSIONS_SEPARATOR));
+        return Stream.of(list.split(VERSIONS_SEPARATOR)).filter(v -> !v.isBlank());
     }
 
     public Boolean isExcluded(String version) {
         // exit early if the version passed in is null or empty
-        if (StringUtils.isBlank(version)) {
+        if (Util.fixEmpty(version) == null) {
             return false;
         }
-        for (String it : getExcludedVersions()) {
-            // confirm that the excluded versions aren't null or empty
-            // and if the version contains the exclusion thus it can be
-            // anywhere in the string.
-            if (StringUtils.isNotBlank(it) && version.contains(it)){
```

Comment:
Was running a redundant null check on result of `String.split`; generally confused. Trying to push input validation farther left; it could be further.

(Also why is this method returning `Boolean` not `boolean`?! Lots of stuff to clean up in the caching code.)